packages/react-components/src/lib/utils/map.ts

```diff
@@ -4,6 +4,10 @@ export const toRadian = (num: number): number => {
   return (num * Math.PI) / 180;
 };
 
+export const toDegrees = (num: number): number => {
```

Comment:
https://github.com/CesiumGS/cesium/blob/1.77/Source/Core/Math.js#L452
https://cesium.com/docs/cesiumjs-ref-doc/Math.html#.toDegrees

Reply:
You are right let's keep it for common reasons OL and CESIUM